### PR TITLE
Add pedestrian tracking example.

### PR DESCRIPTION
**Pull request by yqhok1**
Signed-off-by: York You <573861119@qq.com>

**What this PR does / why we need it**:
Support for #20.
**Which issue(s) this PR fixes**:
https://github.com/kubeedge/sedna/issues/309

To run this example, Sedna should be updated with https://github.com/kubeedge/sedna/pull/375

### CODE REVIEW

File: examples/pedestrian_tracking/generate_reports.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
+import argparse
+import datetime
+import pandas as pd
+from pathlib import Path
+from fpdf import FPDF
+from core.common import utils
+
```
Comment (MooreZheng) [inline examples/pedestrian_tracking/generate_reports.py:7]: This generation code looks good to me. But it is designed to applied on multi-edge inference and might not work for other bench, e.g., federated learning and lifelong learning. Our suggestion is to put the generate_report.py into ./multiedge_inference_bench/testreport.

BTW, the sample_report.pdf can also put in ./multiedge_inference_bench/testreport.

**Review by MooreZheng — COMMENTED** [review]
Overall it looks good to me. A minor suggestion: the generate_report.py and the sample_report.pdf can put in ./multiedge_inference_bench/testreport, as discussed in https://github.com/kubeedge/ianvs/pull/36/files#r1004189534

File: examples/pedestrian_tracking/multiedge_inference_bench/testalgorithms/reid/m3l/m3l_algorithm.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+algorithm:
```
Comment (jaypume) [inline examples/pedestrian_tracking/multiedge_inference_bench/testalgorithms/reid/m3l/m3l_algorithm.yaml:1]: Please add copyright to **every** code file.

File: examples/pedestrian_tracking/multiedge_inference_bench/testalgorithms/reid/m3l/reid/__init__.py [hunk 3]
```diff
@@ -0,0 +1,6 @@
+from __future__ import absolute_import
```
Comment (jaypume) [inline examples/pedestrian_tracking/multiedge_inference_bench/testalgorithms/reid/m3l/reid/__init__.py:1]: Please add copyright to **every** python code file.

File: examples/pedestrian_tracking/multiedge_inference_bench/testalgorithms/reid/m3l/reid/models/IBNMeta.py [hunk 4]
```diff
@@ -0,0 +1,222 @@
+from __future__ import absolute_import
```
Comment (jaypume) [inline examples/pedestrian_tracking/multiedge_inference_bench/testalgorithms/reid/m3l/reid/models/IBNMeta.py:1]: If these code is a copy of opensource repository, please keep the original copyright here.

File: examples/pedestrian_tracking/multiedge_inference_bench/testalgorithms/reid/m3l/reid/__init__.py (hunk 3 above)
Comment (jaypume) [reply examples/pedestrian_tracking/multiedge_inference_bench/testalgorithms/reid/m3l/reid/__init__.py:1]: If the code is developed by you, the copyright should be from "KubeEdge", you can refer to a ianvs code file, like [this](https://github.com/kubeedge/ianvs/blob/main/core/testenvmanager/testenv/testenv.py)

File: examples/pedestrian_tracking/generate_reports.py (hunk 1 above)
Comment (yqhok1) [reply examples/pedestrian_tracking/generate_reports.py]: Thanks for your suggestion, I have moved the code to the correct position.

File: examples/pedestrian_tracking/multiedge_inference_bench/testalgorithms/reid/m3l/reid/__init__.py (hunk 3 above)
Comment (yqhok1) [reply examples/pedestrian_tracking/multiedge_inference_bench/testalgorithms/reid/m3l/reid/__init__.py:15]: Thanks for your suggestion, I have added the copyright to every code file.

**Pull request merged**